timemachine/fe/interpolate.py

```diff
@@ -14,6 +14,10 @@ class DuplicateAlignmentKeysError(RuntimeError):
 Key = Any
 
 
+def to_hashable(x):
```

Comment:
this was moved out so it shows up in profiling.

Reply:
Aside: I think we might be able to just delete this in a future PR; IIRC it exists to deal with some inconsistencies in the types and number of dimensions of the index tuples for different potentials.